releases/Process/Major Release Process.md

```diff
@@ -70,16 +70,31 @@ Entry Criteria: Release SIG has has indicated the need for a release and has a g
 1. **Release SIG** coordinates with the Technical Steering Committee, the Open 3D Foundation Marketing Committee, and the SIGs to determine a release date. This may be done by the SIG chair, co-chair, or the release manager if one has been chosen.
  2. **Release Manager** begins the feature list (see below)
  3. **Release Manager** Creates Milestones for each of the repositories outlined in [Key Details](#key-details). Here is an example from the 22.10.0 Release. https://github.com/o3de/o3de/milestone/11 - follow a similar naming convention.
- 4.  **Release Manager** creates the Key Information Github Issue similar to https://github.com/o3de/sig-release/issues/64. The releaser manager should maintain the Key Information throughout the release. It should be updated as more information about the release is known, or when details change. The release manager should also post a link to the Key Information in #sig-all Discord channel to inform the community of the upcoming release, If the Key Information changes, the Release Manager should post a new message to #sig-all noting that release information has changed.
- 5. **Release Manager** creates a project board for the release with all tasks in the O3DE Repository. Refer to the project board and tasks from the previous release, then manually create tasks in your new release  1-by-1 based on the previous release's tasks. Previous project boards are located at https://github.com/o3de/o3de/projects?type=classic
- 6. **Release Manager** ensures all Planning tasks on the project board are completed, either by completing them or assigning them to others.
- 7. **Release Manager**  has a Planning task on the project board to identify all Release Roles. These release roles should be updated in the Key Information once known.
+ 4.  **Release Manager** creates the Key Information 
```

Comment:
Do we need to give guidance on how long each critical step in the release timeline lasts, e.g. it's recommended for the Stabilization period to run for a month? Or do you think giving the template is enough?

Reply:
I have changed this to suggest using the previous release key information. I think this is best because we make adjustments each release and the last release is probably the latest and greatest estimate of how long each phase will take. It's not perfect, but I think it's the best we can do for now.